cms/__init__.py

```diff
@@ -137,6 +139,20 @@
     LANG_PASCAL: ".o",
 }
 
+COMPILED = 1
+HALF_COMPILED = 3
+INTERPRETED = 5
```

Comment:
Oh I see. My bad; I also misunderstood.

So we're going to need to have at least one python solution per problem anyway as a smoke test -- just use that as a blind multiplier and it'll be good enough for now, I guess. Stick in a TODO and assign me the issue, if you can; I have some ideas on how we can generally try and set these a little better which I'll come back to ~~one day~~